Library/Homebrew/cask/lib/hbc/cli/reinstall.rb

```diff
@@ -0,0 +1,51 @@
+module Hbc
+  class CLI
+    class Reinstall < Install
+
+      def self.install_casks(cask_tokens, force, skip_cask_deps, require_sha)
+        count = 0
+        cask_tokens.each do |cask_token|
+          begin
+            cask = Hbc.load(cask_token)
+
+            latest_installed_version = cask.timestamped_versions.last
+
+            unless latest_installed_version.nil?
+              latest_installed_cask_file = cask.metadata_master_container_path
+                                               .join(latest_installed_version.join(File::Separator),
+                                               "Casks", "#{cask_token}.rb")
+
+              # use the same cask file that was used for installation, if possible
+              cask = Hbc.load(latest_installed_cask_file) if latest_installed_cask_file.exist?
+            end
+
+            # Always force uninstallation, ignore method parameter
+            Installer.new(cask, force: true).uninstall
+            Installer.new(cask,
+                               force:          force,
+                               skip_cask_deps: skip_cask_deps,
+                               require_sha:    require_sha).install
+            count += 1
+          rescue CaskAlreadyInstalledError => e
```

Comment:
Same thing here, instead of duplicating `CLI::Install`, simply call `CLI::Install::run`.